client/src/Validation/validation.js

```
@@ -278,35 +301,56 @@ const logoImageValidation = joi
+	color: joi.string().trim().messages({
+		"string.empty": "Color is required.",
 	}),
+
+	monitorSelectionMode: joi
+		.string()
+		.valid("manual", "cidr")
+		.default("manual"),
+
+	cidrRanges: joi.when("monitorSelectionMode", {
+		is: "cidr",
+		then: joi.array().items(cidrSchema).min(1).required(),
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)